R/request.R

```diff
@@ -99,7 +99,14 @@ print.request <- function(x, ...) {
 
 request_prepare <- function(req) {
   req <- request_combine(request_default(), req)
-  req$options$customrequest <- req$method
+  switch(req$method,
+         "POST" = {
```

Comment:
Yes, but it really doesn't matter here. Regardless, splitting the expression over multiple lines really bugs me, so please fix.